cinema/views.py

```diff
@@ -158,6 +178,23 @@ def get_queryset(self):
 
         return queryset
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "data",
+                type=OpenApiTypes.DATE,
+                description="Filter by data",
```

Comment:
The OpenAPI parameter description should be 'Filter by date' instead of 'Filter by data' to accurately describe the filter parameter for the movie session list.